tests/v2/validation/rbac/rbac_test.go

```diff
@@ -344,7 +344,7 @@ func (rb *RBTestSuite) TestRBAC() {
 	}
 	for _, tt := range tests {
 		rb.Run("Set up User with Cluster Role "+tt.name, func() {
-			newUser, err := createUser(rb.client)
+			newUser, err := createUser(rb.client, "user")
```

Comment:
I think adding them hard coded is not future-proof like I mentioned above for the deployment, either a global variable or function-scoped variable might be better for future updates for these values.

Reply:
Updated. Added the user as a constant in rbac.go file.